qwik-graphql-tailwind/src/components/repo-about/homepage-url/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { component$ } from '@builder.io/qwik';
+import { Link } from '@builder.io/qwik-city';
+import { LinkIcon } from '~/components/icons';
+import * as styles from '../repo-about.className';
+
+interface HomepageUrlProps {
+  homepageUrl?: string | null;
```

Comment:
the `?` means it could be undefined or null, so not sure why its there when there is a null among the possible value

Reply:
Most of the interfaces were copied from the other projects.